.github/ISSUE_TEMPLATE/bug-fixes.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Bug fixes
+about: The default template, issues in this repository are for bug fixes.
+title: ''
```

Comment:
Is this a table? What is the use of each column/field?

Reply:
Have you seen this article on issue template builders? https://help.github.com/en/articles/about-issue-and-pull-request-templates#issue-templates

[Issues Templates](https://help.github.com/en/articles/creating-issue-templates-for-your-repository)

Reply:
Yeah, this is sadly the result of that process 😢